static/page-templates/page-templates.json

```diff
@@ -2,84 +2,94 @@
 	"description": "WordPress.com page templates",
 	"templates": [
 		{
-			"title": "rockfield",
-			"defaults": "rockfield",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rockfield2",
-			"defaults": "rockfield",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "leven",
-			"defaults": "leven",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "alves",
-			"defaults": "alves",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "alves2",
-			"defaults": "alves",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "balasana",
-			"defaults": "balasana",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "coutoire",
-			"defaults": "coutoire",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "exford",
-			"defaults": "exford",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rockfield",
-			"defaults": "rockfield",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "stratford",
-			"defaults": "stratford",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "stratford2",
-			"defaults": "stratford",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "mayland",
-			"defaults": "mayland",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "mayland2",
-			"defaults": "mayland",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "maywood",
-			"defaults": "maywood",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "maywood2",
-			"defaults": "maywood",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rivington",
-			"defaults": "rivington",
-			"vertical-whitelist": []
+			"title": "Vesta",
+			"slug": "vesta",
+			"template": "mayland2",
```

Comment:
Is there any chance we can get rid either of the slug, or the template field? (I hope not to misrepresent, but IIRC, Jon got those mixed up earlier today, too.) 

There are a lot of entities here. IIUC, we call Vesta a... design? But I haven't fully figured out why we need to differentiate between that, and a template. Can we not equate designs with templates -- at least here, for engineering purposes? Is the differentiator the font pairing? (If it is, I'd still suggest going with one entity -- template -- and thinking of that font pairing simply as the default option.)

(I think we can still refer to a combination of theme, template, font pairing/global style as a design -- probably using the template name.)

Reply:
>I think we can still refer to a combination of theme, template, font pairing/global style as a design -- probably using the template name.

Is the plan to be able to use one template for multiple "designs"? If so, equating the template name with design could be problematic later down the road.

It seems like any combination of template+theme+global styles/fonts (i.e. "design") should have a unique identifier.

Reply:
> > I think we can still refer to a combination of theme, template, font pairing/global style as a design -- probably using the template name.
> 
> Is the plan to be able to use one template for multiple "designs"? If so, equating the template name with design could be problematic later down the road.

I think we need to clarify which of these are always coupled, and which can be freely exchanged. E.g. I think the plan is for the user to always select any Global Style/Font Pairing with any template/theme (you could say Global Styles/Font Pairing is one degree of freedom). 

> It seems like any combination of template+theme+global styles/fonts (i.e. "design") should have a unique identifier.

But I don't think we'd want individual names for each of 'Rivington+Rockfield+Arvo/Montserrat', 'Rivington+Rockfield+Chivo/Open Sans', etc. That would get unwieldy pretty soon (what's a good name for 'Rivington+Rockfield+Arvo/Montserrat' -- something kinda descriptive such as 'RivRockArMon'? 'RRAM'? Or another random name such as 'Gaudi'?). I think it's fine to refer to each design by naming which element is chosen for each degree of freedom (so, back to 'Rivington+Rockfield+Arvo/Montserrat').

tl;dr -- I wouldn't equate design with template names. We can refer to designs by the theme/template/global style they're made up of.

Reply:
think of a "design" as a pair of "theme + homepage template + presets for global styles". I agree we shouldn't equate design and template names for .... well, the reasons you mention. It doesn't make sense because a template could potentially be reused in several designs. But at the moment, that's what we have, so we'll iterate to a place where those names are not tied